src-core-pp/jbuild

```diff
@@ -0,0 +1,12 @@
+(jbuild_version 1)
+
+(library
+  ( (name        core_pp)
+    (public_name core_pp)
```

Comment:
If you remove `public_name` from these jbuild files you don't have to create opam files for all the private sub-packages. Probably only the root package should have a `public_name`.